src/peft/tuners/adalora.py

```diff
@@ -183,6 +183,9 @@ def _find_and_replace(self, adapter_name):
                         new_module = SVDLinear4bit(
                             adapter_name, target.in_features, target.out_features, bias=bias, **fourbit_kwargs
                         )
+                    elif is_auto_gptq_available() and target.__class__.__name__ == "QuantLinear":
```

Comment:
Can the check not be `and isinstance(target, QuantLinear)` so that it would also work on subclasses?

Reply:
Changed, let me know if it works with you. Also, check the comment above for the reason.